src/pages/user/viewer/details.jsx

```diff
@@ -1,36 +1,102 @@
-import React, { useState, useEffect } from 'react';
-import { useParams } from "react-router-dom";
+import React, { useState, useEffect, useContext } from 'react';
 import styled from 'styled-components';
-import { User } from "@phoenixlan/phoenix.js";
+import { User, Avatar } from "@phoenixlan/phoenix.js";
 import { Table, TableCell, TableHead, SelectableTableRow, TableRow, TableBody } from "../../../components/table";
 import { PageLoading } from '../../../components/pageLoading';
-import { InnerContainer, InnerContainerRow, InnerContainerTitle, InputContainer, InputLabel } from '../../../components/dashboard';
+import { CardContainer, CardContainerIcon, CardContainerInnerIcon, CardContainerInnerText, CardContainerText, InnerContainer, InnerContainerRow, InnerContainerTitle, InputContainer, InputLabel, PanelButton } from '../../../components/dashboard';
 
-import { Button } from "../../../components/button"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar as faStarRegular, faAddressCard, faCalendar, faEnvelope, faUser } from '@fortawesome/free-regular-svg-icons';
+import { faStar as faStarSolid, faCheck, faCode, faFileContract, faMapPin, faMars, faPhone, faPhoneSlash, faPrint, faUserPen, faVenus } from '@fortawesome/free-solid-svg-icons';
+import { Colors } from '../../../theme';
+import { dateOfBirthToAge } from '../../../utils/user';
+import { AuthenticationContext } from '../../../components/authentication';
+import { useHistory } from 'react-router-dom/cjs/react-router-dom.min';
 
 const S = {
     Avatar: styled.img`
         width: 256px;
-        border: 1px solid rgb(235, 235, 235);
+        border: 1px solid ${Colors.Gray200};
+        margin-bottom: 1em;
+
+        @media screen and (max-width: 480px) {
+            width: 100%;
+        }
     `,
 }
 
-export const UserViewerDetails = ({ user }) => {
+export const UserViewerDetails = ({ inheritUser, reloadFunction }) => {
+
+    const [ user, setUser ] = useState(inheritUser);
```

Comment:
no need for useState, just rawdog the `user` function parameter

Reply:
Fixed!